src/constants.js

```diff
@@ -112,3 +113,17 @@ export const INVENTORY_WRITE_PERMISSIONS = [
     'inventory:hosts:write',
     'inventory:*:write'
 ];
+
+export const getSearchParams = () => {
```

Comment:
If I had time, I would do a larger refactoring of filtering in the Inventory App. but for now, this is bearable.